atulkrishnathakur_fastcrud_router_api_file_route_71.py

```
@@ -0,0 +1,38 @@
+        file_location = os.path.join(UPLOAD_DIRECTORY, file.filename)
+        with open(file_location, "wb+") as file_object:
+            file_object.write(file.file.read())
+        file_url = f"http://localhost:8000/uploads/{file.filename}"
+
+        return {"info": f"file '{file.filename}' saved at '{file_location}' url '{file_url}'"}
+    except Exception as e:
+        print(f"Exception error {e}")
+
+
+@router.post("/upload-file-by-uploadfile",name="uploadfile")
+def uploadFile(file:UploadFile):
```

Ruff (0.13.1)

Redefinition of unused `uploadFile` from line 14

(F811)